packages/components/README.md

```diff
@@ -34,3 +36,68 @@ Installation is simple. All you need to do is add `<courier>` components to your
   <script src="https://courier-components-xvdza5.s3.amazonaws.com/latest.js"></script>
 </body>
 ```
+
+## [SDK](#SDK)
+
+To interact with the components, we have exposed an sdk on `window.courierSdk`. Since this sdk initializes asynchronous, you can create and push initialization functions onto an array named `window.courierAsyncInit`.
+
+Example:
+
+```html
+<script type="text/javascript">
+  window.courierConfig = {
+    clientKey: "{{CLIENT_KEY}}",
+    userId: "{{USER_ID}}",
+  };
+
+  window.courierAsyncInit = window.courierAsyncInit || [];
+  window.courierAsyncInit.push(() => {
+    window.courierSdk.toast({
+      title: "Hello",
+      body: "World",
+    });
+
+    window.courierSdk.transport.intercept((message) => {
+      const iWantToHideThisMessage = true;
+      if (iWantToHideThisMessage) {
+        return;
+      }
+
+      // you can also mutate this message before displaying to a user
+      return message;
+    });
+  });
+</script>
+```
+
+## [Configuring Components](#config)
+
+You can configure components in 2 different ways. Inline html attributes but also through `window.courierConfig`.
+
+1. Inline
+
+Configuring simple options through HTML attributes is simple. For each configuration, make sure you are using `kebab-case` https://www.theserverside.com/definition/Kebab-case instead of `camelCase` because HTML attributes are case insensitive.
+
+Example:
+
+```html
+<courier-toast auto-close="false"></courier-toast>
+```
+
+2. `window.courierConfig`
+
+Some components have many options and adding many attributes to HTML entities can be cumbersome, you can also add configuration through our `window.courierConfig` object.
+
+Example:
+
+```javascript
+window.courierConfig = {
+  components: {
+    toast: {
+      autoClose: false,
+    },
+  },
+};
+```
+
+> Inline configuration attributes will take precedence over `window.courierConfig` options
```

Comment:
maaaaybe having this note above before the examples is more helpful.

Reply:
updated